presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -616,6 +636,24 @@ private static DateTimeField getTimestampField(ISOChronology chronology, Slice u
         throw new PrestoException(INVALID_FUNCTION_ARGUMENT, "'" + unitString + "' is not a valid Timestamp field");
     }
 
+    private static long getUnixtimeField(Slice unit)
```

Comment:
Maybe rename something like getSecondsPerUnit. The field is not what's returned here.

Reply:
yes, thank you so much